pkg/sensors/tracing/selectors_test.go

```diff
@@ -223,7 +223,7 @@ func TestTracepointSelectors(t *testing.T) {
 					return nil
 				}
 
-				ret[whence] = ret[whence] + 1
+				ret[whence] += +1
```

Comment:
+= +1 seems weird.

Reply:
here it was rightly fixed https://github.com/cilium/tetragon/pull/983/files/3c105466c9d6eb785f87070ef35704167a6e43bf#diff-6b20c8ee0a0503a91754d733ce9932baa6b15a784384595466f3f05fdc033d5fL353-R353.

Reply:
fixed! thanks